### PR TITLE
fix README - issue #533

### PR DESCRIPTION
**Pull request by brianhays**
fix for #533

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ We teach:
 * Javascript and Jquery
 * How to go about getting a job in the industry
 
-This repo contains the main app for the Odin Project which can be found online at [theodinproject.com](theodinproject.com). It pulls in lesson and project content from our [curriculum repo](https://github.com/TheOdinProject/curriculum).
+This repo contains the main app for the Odin Project which can be found online at [theodinproject.com](http://theodinproject.com). It pulls in lesson and project content from our [curriculum repo](https://github.com/TheOdinProject/curriculum).
```
Comment (codyloyd) [inline README.md]: make it https

Comment (brianhays) [inline README.md]: can do!

Comment (brianhays) [inline README.md]: Actually @codyloyd https on the root domain times out. The current link (http://theodinproject.com) redirects to https://www.theodinproject.com 

I'll change the link to https://www.theodinproject.com

**Comment by brianhays** [comment]
link updated, also updated the header link with the same URL

File: README.md (hunk 1 above)
Comment (codyloyd) [inline README.md]: that's fine

**Pull request merged**